docs/src/main/sphinx/connector/exasol.md

```diff
@@ -0,0 +1,210 @@
+---
+myst:
+  substitutions:
+    default_domain_compaction_threshold: '`32`'
+---
+
+# Exasol connector
+
+```{raw} html
+<img src="../_static/img/exasol.png" class="connector-logo">
+```
+
+The Exasol connector allows querying an [Exasol](https://www.exasol.com/) database.
+
+## Requirements
+
+To connect to Exasol, you need:
+
+* Exasol database version 7.1 or higher.
+* Network access from the Trino coordinator and workers to Exasol.
+  Port 8563 is the default port.
+
+## Configuration
+
+To configure the Exasol connector as the ``example`` catalog, create a file
+named ``example.properties`` in ``etc/catalog``. Include the following
+connection properties in the file:
+
+```text
+connector.name=exasol
+connection-url=jdbc:exa:exasol.example.com:8563
+connection-user=user
+connection-password=secret
+```
+
+The ``connection-url`` defines the connection information and parameters to pass
+to the JDBC driver. See the
+[Exasol JDBC driver documentation](https://docs.exasol.com/db/latest/connect_exasol/drivers/jdbc.htm#ExasolURL)
+for more information.
+
+The ``connection-user`` and ``connection-password`` are typically required and
+determine the user credentials for the connection, often a service user. You can
+use {doc}`secrets </security/secrets>` to avoid using actual values in catalog
+properties files.
+
+:::{note}
+If your Exasol database uses a self-signed TLS certificate you will must
+specify the certificate's fingerprint in the JDBC URL using parameter
+``fingerprint``, e.g.: ``jdbc:exa:exasol.example.com:8563;fingerprint=ABC123``.
+:::
+
+```{include} jdbc-authentication.fragment
+```
+
+```{include} jdbc-common-configurations.fragment
+```
+
+```{include} jdbc-domain-compaction-threshold.fragment
+```
+
+```{include} jdbc-case-insensitive-matching.fragment
+```
+
+(exasol-type-mapping)=
+
+## Type mapping
+
+Because Trino and Exasol each support types that the other does not, this
+connector {ref}`modifies some types <type-mapping-overview>` when reading or
+writing data. Data types may not map the same way in both directions between
+Trino and the data source. Refer to the following sections for type mapping in
+each direction.
+
+### Exasol to Trino type mapping
+
+Trino supports selecting Exasol database types. This table shows the Exasol to
+Trino data type mapping:
+
+```{eval-rst}
+.. list-table:: Exasol to Trino type mapping
+  :widths: 30, 25, 50
+  :header-rows: 1
+
+  * - Exasol database type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DECIMAL(p, s)``
+    - ``DECIMAL(p, s)``
+    -  See :ref:`exasol-number-mapping`
+  * - ``DOUBLE PRECISION``
+    - ``REAL``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
```

Comment:
Please reorder as boolean, double, decimal, char, varchar, date. I would recommend removing timestamp from this PR.